openreview/venue/matching.py

```diff
@@ -275,6 +284,18 @@ def _build_note_conflicts(self, submissions, user_profiles, get_profile_info):
                 conflicts.update(author_emails.intersection(user_info['emails']))
                 conflicts.update(author_publications.intersection(user_info['publications']))
 
+                ## Transfer SAC conflicts
+                if len(conflicts) == 0 and self.is_area_chair:
```

Comment:
if self.is_area_chair, why would len(conflicts)==0?

Reply:
because if there are already conflicts between the AC and the paper we don't need to compute the SAC conflicts.